src/main.ts

```diff
@@ -370,7 +374,7 @@ class SettingsTab extends PluginSettingTab {
       const sysLocale = navigator.language?.toLowerCase();
 
       const localizedWeekdays = window.moment.weekdays();
-      const localeWeekStartNum = window._bundledLocaleWeekSpec.dow;
+      const localeWeekStartNum = window._bundledLocaleWeekSpec?.dow;
```

Comment:
This change really shouldn't be necessary but I figured it doesn't hurt to be extra defensive here.